File: AsalSayiBulma/AsalSayiBulma.java
package asalsayibulma;

import java.util.Scanner;

public class AsalSayiBulma {

    public static void main(String[] args) {
        Scanner reader = new Scanner(System.in);
        System.out.print("Lütfen pozitif bir tamsayı girin: ");
        int sayi = reader.nextInt();
        boolean kontrol;
        if (sayi > 2) {
            System.out.print("2");
        }
        for (int i = 3; i < sayi; i++) {
            kontrol = false;
            for (int j = 2; j < i; j++) {
                if (i % j == 0) {
                    kontrol = true;
                    break;
                }
            }
            if (kontrol == false) {
                System.out.print("," + i);
            }
        }
        System.out.println("");
    }
}

File: AsalSayiBulma/AsalSayiBulma.py
def asal_sayilari_yazdir(sayi):
    for i in range(2, sayi+1):
        halen_asal = True
        for j in range(2, i):
            if i % j == 0:
                halen_asal = False
                break
        if halen_asal == True:
            print(i, end=", ")

asal_sayilari_yazdir(int(input("Bir sayı girin : ")))